spec/features/merchants/merchant_index_spec.rb

```diff
@@ -87,5 +87,35 @@
         expect(page).to have_button("Enable Merchant")
       end
     end
+
+    context "as an admin merchant" do
```

Comment:
```suggestion
```
You're still within the "as an admin user" describe block -- you don't need this, and there are no "admin merchants" -- confusing

Reply:
Will have to delete the corresponding `end` too.